test/ReCaptchaTest.php

```diff
@@ -257,30 +257,22 @@ public function testVerifyWithMissingSecretKey()
     {
         $this->expectException('ZendService\\ReCaptcha\\Exception');
 
-        $this->reCaptcha->verify('challenge', 'response');
+        $this->reCaptcha->verify('response');
     }
 
     public function testVerifyWithMissingIp()
     {
         $this->expectException('ZendService\\ReCaptcha\\Exception');
 
         $this->reCaptcha->setSecretKey($this->secretKey);
-        $this->reCaptcha->verify('challenge', 'response');
-    }
-
-    public function testVerifyWithMissingChallengeField()
-    {
-        $this->reCaptcha->setSecretKey($this->secretKey);
-        $this->reCaptcha->setIp('127.0.0.1');
-        $response = $this->reCaptcha->verify('', 'response');
-        $this->assertFalse($response->getStatus());
+        $this->reCaptcha->verify('response');
     }
 
     public function testVerifyWithMissingResponseField()
     {
         $this->reCaptcha->setSecretKey($this->secretKey);
         $this->reCaptcha->setIp('127.0.0.1');
-        $response = $this->reCaptcha->verify('challenge', '');
+        $response = $this->reCaptcha->verify(null);
```

Comment:
This test is still failing, doesn't matter what I set here always I'm not getting false status response.
I wonder how it passed with v3 release?

Reply:
I've done some research, and it appears that payloads to the API verification point expect:

```json
{
    "secret": "secret key",
    "remoteip": "optional; IP address of user",
    "response": "required: user response token verifying user on your site"
}
```

What's interesting is that while "response" is required, it does not appear that the ReCAPTCHA API cares what the value is, just so long as it is present. As such, this test is essentially moot; it will _always_ return with a `"success": true` so long as the API key is valid.

I'll remove this test during merge.